apps/server-web/src/main/menu.ts

```
@@ -143,24 +135,38 @@ export default class MenuBuilder {
+        return this.initialMenu();
+      default:
+        return this.defaultMenu();
     }
-    return menuCopied;
-  })
-}
+  }
+
+  buildTemplateMenu(windowType: IWindowTypes, i18nextMainBackend: typeof i18n) {
+    const menu = this.getWindowMenu(windowType)
+    return Menu.buildFromTemplate(this.translateAppMenu(i18nextMainBackend, menu));;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)